pkg/plugin/cli/trait.go

```diff
@@ -0,0 +1,46 @@
+/*
+ Copyright 2021. The KubeVela Authors.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package cli
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/oam-dev/kubevela/apis/types"
+	"github.com/oam-dev/kubevela/pkg/utils/common"
+	cmdutil "github.com/oam-dev/kubevela/pkg/utils/util"
+	"github.com/oam-dev/kubevela/references/cli"
+)
+
+// NewTraitCommand creates `trait` command
+func NewTraitCommand(c common.Args, ioStreams cmdutil.IOStreams) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:                   "trait",
+		DisableFlagsInUseLine: true,
+		Short:                 "Show traits in cap center",
+		Long:                  "Show traits in cap center",
+		Example:               "kubectl vela trait",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			err := cli.PrintDefaultCapTraitList(ioStreams)
+			return err
+		},
+		Annotations: map[string]string{
+			types.TagCommandType: types.TypeCap,
```

Comment:
```suggestion
			types.TagCommandType: types.TypePlugin,
```
The functions provided by the kubectl plugin should all be set to `TypePlugin`, maybe you need to modify the value of TypePlugin.

Reply:
I changed the discription. Maybe one Tag for all the plugin isn't enough in future.